dev/com.ibm.websphere.appserver.thirdparty.opentracing/bnd.bnd

```diff
@@ -13,9 +13,6 @@ bVersion=1.0
 
 instrument.disabled: true
 
--includeresource: \
```

Comment:
Looks good, but can you verify that the websphere.appserver.thirdparty.opentracing api/spi JAR contains the same classes? I can't tell 100% from the bnd that it would.

Reply:
```
$ jar tvf dev/build.image/wlp/dev/api/third-party/com.ibm.websphere.appserver.thirdparty.opentracing_1.0.21.jar | grep io/opentracing/.*class | wc -l
20
```